logger/logger.go

```diff
@@ -8,13 +8,14 @@ import (
 	"sync"
 	"time"
 
+	"code.cloudfoundry.org/lager/v3"
 	"go.uber.org/zap"
 	"go.uber.org/zap/exp/zapslog"
 	"go.uber.org/zap/zapcore"
 )
 
 var (
-	conf        dynamicLoggingConfig
+	Conf        DynamicLoggingConfig
```

Comment:
why is this exported now? I assume for the change in main.go? 

exposing the variable means anyone anywhere can change it. what about adding a getter function instead?

Reply:
We need the satisfy the debugserver's reconfigurable sink: https://github.com/cloudfoundry/debugserver/blob/main/server.go#L24-L26

The [base gorouter logger is inited at the package level and privately](https://github.com/cloudfoundry/gorouter/blob/main/logger/logger.go#L67-L69) puts us in a bit of an odd spot with further changes to the logger. I exported the config as we need the `debugserver` to actually change the config via `SetMinLevel()`

Basically since the `grlogger.baseLogger` is [initialized on importation via `init()`](https://github.com/cloudfoundry/gorouter/blob/main/logger/logger.go#L67-L69)  and private to the package, I pushed the config out as public as we need something to satisfy the `ReconfigurableSinkInterface`.  A getter doesn't quite fit the pattern. There is an argument for pushing it further down, but this was a bit simpler.

I went with the config as it's the area where adding the method to satisfy the `ReconfigurableSinkInterface` was the least intrusive.